### PR TITLE
add header bar at top, clicking on sjc logo links to root path

### PR DESCRIPTION
**Pull request by michaeldo2**
### What does this PR do?
creates a header bar that displays sjc logo and name of current user

### Status
READy

### Migrations
No

### Related PRs
None

### Screenshots
screen shot 2017-11-23 at 1 42 56 am

CC: @calvind777

### CODE REVIEW

File: app/assets/javascripts/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Header extends React.Component {
+	render() {
+		const { user, imagePath } = this.props;
+		const { Image } = ReactBootstrap;
+		return (
+			<div className="sjc-header">
```
Comment (calvind777) [inline app/assets/javascripts/components/Header.jsx]: these tabs bro LOOOOOL

Comment (calvind777) [reply app/assets/javascripts/components/Header.jsx]: it's like 8 spaces indented idk ;.;

Comment (michaeldo2) [reply app/assets/javascripts/components/Header.jsx]: LOOOOL god jesus christ im such a scrub

Comment (michaeldo2) [reply app/assets/javascripts/components/Header.jsx]: fixed T___T

**Pull request merged**